README.md

```diff
@@ -25,6 +25,7 @@ The config values to be set are:
 | `NAMESPACE`              | Namespace where daemonset is to be created | `kubernetes-image-puller` |
 | `IMAGES`                 | List of images to be cached, in the format `<name>=<image>;...` | Contains a default list of images, but should be configured when deploying |
 | `NODE_SELECTOR` | Node selector applied to pods created by the daemonset       | `'{}'` |
+| `IMAGE_PULL_SECRETS` | List of image pull secrets, in the format `pullsecret1;...` to add to pods created by the daemonset       | `""` |
```

Comment:
is it expected that the `pullsecret1` is created by the cluster admin? should it be created in the very same namespace where the image puller is installed?

Reply:
Yep exactly, secrets are required to be in the same namespace and to be created by the admin

Reply:
Done